rosdep/python.yaml

```
@@ -6390,6 +6390,12 @@ python3-future:
   openembedded: [python3-future@meta-python]
   rhel: ['python%{python3_pkgversion}-future']
   ubuntu: [python3-future]
+python3-gdal:
+  debian: [python3-gdal]
+  fedora: [python3-gdal]
+  opensuse: [python3-GDAL]
+  rhel: [python3-gdal]
```

Hmm now the checks are complaining it doesn't exist in RHEL 7.
https://access.redhat.com/discussions/5354171
It's only packaged via EPEL. Maybe the EPEL packages don't count. @cottsay does that mean it's not really available on RHEL? We followed the GitHub Actions earlier that suggested it may be available on RHEL, and now it says it isn't.

---

The GitHub Action makes suggestions based off from the latest version of each distribution, which for RHEL is RHEL 8. We would need to improve the suggestion system to make more specific suggestions and check all supported versions for each version of a distribution - something we don't currently have any resources to develop.

EPEL is an accepted source for RHEL rules in rosdep, but the set of available packages varies by major release, so it's not uncommon to see a package available for RHEL 8 but not RHEL 7 (and vice versa).

```suggestion
  rhel:
    '*': [python3-gdal]
    '7': null
```

---

For my future reference, is the EPEL version number the same as the RHEL version number? I'm seeing on this page that there is EPEL 7 and 8 for this package https://src.fedoraproject.org/rpms/gdal But I guess that's not the same as being available in RHEL 7 and 8?